tls/s2n_send.c

```diff
@@ -181,3 +218,20 @@ ssize_t s2n_send(struct s2n_connection * conn, const void *buf, ssize_t size, s2
 
     return size;
 }
+
+#if S2N_GCC_VERSION_AT_LEAST(4,6,0)
+#pragma GCC diagnostic push
+#endif
+#pragma GCC diagnostic ignored "-Wcast-qual"
+ssize_t s2n_send(struct s2n_connection * conn, const void *buf, ssize_t size, s2n_blocked_status * blocked)
```

Comment:
Why do we need these compiler suppressions now? We didn't before and the function signature hasn't changed.

Reply:
This is because iov uses (void*) and parameter in is (const void*).